config/reader.go

```diff
@@ -621,7 +619,12 @@ func getFromCloudOrCache(ctx context.Context, cloudCfg *Cloud, shouldReadFromCac
 				// return cache err
 				return nil, cached, cacheErr
 			}
-			logger.Warnw("unable to get cloud config; using cached version", "error", err)
+
+			lastUpdated := "unknown"
+			if fInfo, err := os.Stat(getCloudCacheFilePath(cloudCfg.ID)); err == nil {
+				lastUpdated = fInfo.ModTime().Format(time.RFC3339)
+			}
+			logger.Warnw("unable to get cloud config; using cached version", "config last updated", lastUpdated, "error", err)
```

Comment:
Nice, last updated is helpful. What does this log look like? /Why are we using that RFC3339 format?

Reply:
sample log in description, time is formatted like`"config last updated":"2024-03-25T14:50:23-04:00"`. RFC3339 just happens to be what we seem to be using in other areas of the codebase (mostly in the CLI). happy to change if you have suggestions

Reply:
sure, updated